src/registrar/assets/sass/_theme/_buttons.scss

```diff
@@ -211,6 +211,10 @@ a.usa-button--unstyled:visited {
     align-items: center;
 }
 
+.usa-button--justify-right {
+  justify-content: right;
+}
+
```

Comment:
I'm going to need some clarification on this...maybe it's because I waited until my "night owl shift", but my brain is just not sure what you're specifying here...